devops/containers/ubuntu2204_preinstalled.Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG base_tag=latest
```

Comment:
Please, update nightly job to produce this image.

Reply:
I'd like to separate that next step from this PR.

First I need to make sure base/build/intel_drivers containers can be built and pushed successfully on schedule. I have a problem pushing these from pull_request task. These are basic containers that are used everywhere.

Then I can update nightly to actually use these containers and run testing on them, we'll need to see results and if they all look good. Nightly containers push depends on successful testing results.
Let me actually cut *_preinstalled.Dockerfile from this PR, as it will be needed on the next mentioned step, not on the first one.